ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -5181,7 +5183,7 @@ let vm_import fd printer rpc session_id params =
               let open Xmlrpc_client in
               let transport =
                 SSL
-                  ( SSL.make ~use_stunnel_cache:true
+                  ( SSL.make ~use_stunnel_cache:true ~verify_cert:None
```

Comment:
This code path imports a zurich / geneva XVA. I don't think we can do much here certificate-wise, and I don't know how used this use-case is. I remember @robhoes saying this code should just be dropped.